components/spi/config.yaml

```diff
@@ -0,0 +1,6 @@
+sharedSecret: <shared-secret>
```

Comment:
what is sharedSecret? That could be just random string, right?

Reply:
yes, any random string